bin/coreos-install

```diff
@@ -301,6 +301,14 @@ fi
 if [[ -z "${BASE_URL}" ]]; then
     BASE_URL="https://${CHANNEL_ID}.release.core-os.net/amd64-usr"
 fi
+
+# if the version is "current", resolve the actual version number
+if [[ "${VERSION_ID}" == "current" ]]; then
+    VERSION_ID=$(wget --inet4-only --quiet -O - "${BASE_URL}/${VERSION_ID}/version.txt" | \
```

Comment:
While we are here let's drop the use of v4 only. That was added ages ago when some v6 server at Google was reporting errors for a few days.